io/disk/ssd/nvmetest.py

```diff
@@ -46,6 +46,10 @@ def setUp(self):
         nvme_node = self.params.get('device', default=None)
         if not nvme_node:
             self.cancel("Please provide valid nvme node name")
+        elif "subsys" in nvme_node:
```

Comment:
@maramsmurthy why subsys name will be there is node_name, it is just kind of /dev/nvmeX right?

Reply:
Hi naresh,
Now we can provide as follows for "device" parameter in yaml file
NQN: nqn.1994-11.com.vendor:nvme:modela:2.5-inch:SERIALNUM
Subsystem: nvme-subsysX
controller: nvmeX
So the nvme_node might have value like "nvme-subsysX"

Reply:
@maramsmurthy can you please add these in the form command above the disk parameter, so that it is clear to any new user